packages/protocol/lib/compatibility/ignored-contracts-v9.ts

```diff
@@ -1,21 +1,29 @@
 export const ignoredContractsV9 = [
   // These are Mento contracts which we are no longer maintaining
   'Exchange',
-  'ExchangeBRL',
   'ExchangeEUR',
+  'ExchangeBRL',
   'GrandaMento',
-  'Reserve',
-  'ReserveSpenderMultiSig',
-  'SortedOracles',
   'StableToken',
-  'StableTokenBRL',
   'StableTokenEUR',
+  'StableTokenBRL',
   'StableTokenRegistry',
+  'Reserve',
+  'ReserveSpenderMultiSig',
+
+  'SortedOracles' 
+  // Note: Sorted Oracles ia a Celo Core Contract
```

Comment:
nit ```suggestion
  // Note: Sorted Oracles is a Celo Core Contract
```

also not sure if we need the last but in the following sentence.